simulator/simulator.cpp

```diff
@@ -64,6 +65,15 @@ int main(int argc, char* argv[]) {
         } else if (strcmp(argv[i], "--smallfield") == 0) {
             Field_Dimensions::Current_Dimensions =
                 Field_Dimensions::Single_Field_Dimensions * scaling;
+        } else if (strcmp(argv[i], "--timeout") == 0) {
+            if (i < argc - 1) {
+                timeoutsimulator = RJ::Seconds(stoi(argv[i + 1]));
```

Comment:
So sorry for not catching this earlier (this is kind of my bad), but stoi can throw an exception if we run 
`./run/simulator --timeout badinput`. 

Can you add a try catch around this, to print out an error message and print usage (or at least exit cleanly).

It seems like the error is `std::invalid_argument`